extraTs/patterns/equilateral.ts

```diff
@@ -0,0 +1,14 @@
+export default function equilateral (a) {
```

Comment:
return type of function and argument 'a' type is missing.

Reply:
Changes Done.